runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -462,12 +462,7 @@ J9::CodeGenerator::lowerCompressedRefs(
       vcount_t visitCount,
       TR_BitVector *childrenToBeLowered)
    {
-   bool isLowMem = false;
-   if (TR::Compiler->vm.heapBaseAddress() == 0)
-      {
-      isLowMem = true;
-      }
-
+   bool isLowMem = true;
```

Comment:
Can we fold uses of this as true?

Reply:
This is actually used later on in this method. It can potentially be redefined on line 637 below:
```
   static char *pEnv = feGetEnv("TR_disableLowHeapMem");
   if (pEnv)
      isLowMem = false;
```

This can influence whether certain branches are taken (not sure if we will ever use this environment variable option though? I don't think we will). Given all that, I've moved this definition closer to its first use for now. If you think it can be removed altogether then let me know and I'll do that.

Reply:
I think we can remove it. There is not "disabling of low heap memory" anymore as it is always 0 now. So whatever code was dependent on that is now dead.